document/js-api/index.bs

```diff
@@ -1149,27 +1143,42 @@ The algorithm <dfn>ToWebAssemblyValue</dfn>(|v|, |type|) coerces a JavaScript va
             1. Return [=ref.null=] |heaptype|.
         1. Otherwise,
             1. Throw a {{TypeError}}.
-    1. If |type| is a subtype of [=ref=] |null| [=heap-type/func=],
+    1. If |heaptype| is a subtype of [=heap-type/func=],
```

Comment:
Should we use the "matches" terminology to match the core spec?

Reply:
Done.